Crash when logging a trackable (related to gallery)

### Describe your problem!

Several beta users report a crash when trying to log trackables.
I can reproduce this with 2022.05.13-RC and got the following log:

```
E/AndroidRuntime(14628): FATAL EXCEPTION: main
E/AndroidRuntime(14628): Process: cgeo.geocaching, PID: 14628
E/AndroidRuntime(14628): java.lang.RuntimeException: Failure delivering result ResultInfo{who=null, request=1, result=0, data=null} to activity {cgeo.geocaching/cgeo.geocaching.TrackableActivity}: java.lang.NullPointerException: Attempt to invoke virtual method 'boolean cgeo.geocaching.ui.ImageGalleryView.onActivityResult(int, int, android.content.Intent)' on a null object reference
E/AndroidRuntime(14628): 	at android.app.ActivityThread.deliverResults(ActivityThread.java:3537)
E/AndroidRuntime(14628): 	at android.app.ActivityThread.handleSendResult(ActivityThread.java:3580)
E/AndroidRuntime(14628): 	at android.app.ActivityThread.access$1300(ActivityThread.java:144)
E/AndroidRuntime(14628): 	at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1327)
E/AndroidRuntime(14628): 	at android.os.Handler.dispatchMessage(Handler.java:102)
E/AndroidRuntime(14628): 	at android.os.Looper.loop(Looper.java:135)
E/AndroidRuntime(14628): 	at android.app.ActivityThread.main(ActivityThread.java:5219)
E/AndroidRuntime(14628): 	at java.lang.reflect.Method.invoke(Native Method)
E/AndroidRuntime(14628): 	at java.lang.reflect.Method.invoke(Method.java:372)
E/AndroidRuntime(14628): 	at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:898)
E/AndroidRuntime(14628): 	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:693)
E/AndroidRuntime(14628): Caused by: java.lang.NullPointerException: Attempt to invoke virtual method 'boolean cgeo.geocaching.ui.ImageGalleryView.onActivityResult(int, int, android.content.Intent)' on a null object reference
E/AndroidRuntime(14628): 	at cgeo.geocaching.TrackableActivity.onActivityResult(TrackableActivity.java:739)
E/AndroidRuntime(14628): 	at android.app.Activity.dispatchActivityResult(Activity.java:6178)
E/AndroidRuntime(14628): 	at android.app.ActivityThread.deliverResults(ActivityThread.java:3533)
E/AndroidRuntime(14628): 	... 10 more
```



### How to reproduce?

- Search for a tracking code
- Post a discover log


### Actual result after these steps?

Crash

### Expected result after these steps?

no crash

### Reproducible

Yes

### c:geo Version

2022.05.13-RC

### System information

_No response_

### Additional Information

_No response_

Taken from users log file in case its any different:
```
05-14 16:37:16.474 28281 28281 E AndroidRuntime: FATAL EXCEPTION: main
05-14 16:37:16.474 28281 28281 E AndroidRuntime: Process: cgeo.geocaching, PID: 28281
05-14 16:37:16.474 28281 28281 E AndroidRuntime: java.lang.RuntimeException: Failure delivering result ResultInfo{who=null, request=1, result=0, data=null} to activity {cgeo.geocaching/cgeo.geocaching.TrackableActivity}: java.lang.NullPointerException: Attempt to invoke virtual method 'boolean cgeo.geocaching.ui.ImageGalleryView.onActivityResult(int, int, android.content.Intent)' on a null object reference
05-14 16:37:16.474 28281 28281 E AndroidRuntime: 	at android.app.ActivityThread.deliverResults(ActivityThread.java:5864)
05-14 16:37:16.474 28281 28281 E AndroidRuntime: 	at android.app.ActivityThread.handleSendResult(ActivityThread.java:5903)
05-14 16:37:16.474 28281 28281 E AndroidRuntime: 	at android.app.servertransaction.ActivityResultItem.execute(ActivityResultItem.java:54)
05-14 16:37:16.474 28281 28281 E AndroidRuntime: 	at android.app.servertransaction.ActivityTransactionItem.execute(ActivityTransactionItem.java:45)
05-14 16:37:16.474 28281 28281 E AndroidRuntime: 	at android.app.servertransaction.TransactionExecutor.executeCallbacks(TransactionExecutor.java:135)
05-14 16:37:16.474 28281 28281 E AndroidRuntime: 	at android.app.servertransaction.TransactionExecutor.execute(TransactionExecutor.java:95)
05-14 16:37:16.474 28281 28281 E AndroidRuntime: 	at android.app.ActivityThread$H.handleMessage(ActivityThread.java:2438)
05-14 16:37:16.474 28281 28281 E AndroidRuntime: 	at android.os.Handler.dispatchMessage(Handler.java:106)
05-14 16:37:16.474 28281 28281 E AndroidRuntime: 	at android.os.Looper.loopOnce(Looper.java:226)
05-14 16:37:16.474 28281 28281 E AndroidRuntime: 	at android.os.Looper.loop(Looper.java:313)
05-14 16:37:16.474 28281 28281 E AndroidRuntime: 	at android.app.ActivityThread.main(ActivityThread.java:8663)
05-14 16:37:16.474 28281 28281 E AndroidRuntime: 	at java.lang.reflect.Method.invoke(Native Method)
05-14 16:37:16.474 28281 28281 E AndroidRuntime: 	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:567)
05-14 16:37:16.474 28281 28281 E AndroidRuntime: 	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1135)
05-14 16:37:16.474 28281 28281 E AndroidRuntime: Caused by: java.lang.NullPointerException: Attempt to invoke virtual method 'boolean cgeo.geocaching.ui.ImageGalleryView.onActivityResult(int, int, android.content.Intent)' on a null object reference
05-14 16:37:16.474 28281 28281 E AndroidRuntime: 	at cgeo.geocaching.TrackableActivity.onActivityResult(TrackableActivity.java:739)
05-14 16:37:16.474 28281 28281 E AndroidRuntime: 	at android.app.Activity.dispatchActivityResult(Activity.java:8659)
05-14 16:37:16.474 28281 28281 E AndroidRuntime: 	at android.app.ActivityThread.deliverResults(ActivityThread.java:5857)
05-14 16:37:16.474 28281 28281 E AndroidRuntime: 	... 13 more
05-14 18:13:17.572 24318 24318 E AndroidRuntime: FATAL EXCEPTION: main
05-14 18:13:17.572 24318 24318 E AndroidRuntime: Process: cgeo.geocaching, PID: 24318
05-14 18:13:17.572 24318 24318 E AndroidRuntime: java.lang.RuntimeException: Failure delivering result ResultInfo{who=null, request=1, result=0, data=null} to activity {cgeo.geocaching/cgeo.geocaching.TrackableActivity}: java.lang.NullPointerException: Attempt to invoke virtual method 'boolean cgeo.geocaching.ui.ImageGalleryView.onActivityResult(int, int, android.content.Intent)' on a null object reference
05-14 18:13:17.572 24318 24318 E AndroidRuntime: 	at android.app.ActivityThread.deliverResults(ActivityThread.java:5864)
05-14 18:13:17.572 24318 24318 E AndroidRuntime: 	at android.app.ActivityThread.handleSendResult(ActivityThread.java:5903)
05-14 18:13:17.572 24318 24318 E AndroidRuntime: 	at android.app.servertransaction.ActivityResultItem.execute(ActivityResultItem.java:54)
05-14 18:13:17.572 24318 24318 E AndroidRuntime: 	at android.app.servertransaction.ActivityTransactionItem.execute(ActivityTransactionItem.java:45)
05-14 18:13:17.572 24318 24318 E AndroidRuntime: 	at android.app.servertransaction.TransactionExecutor.executeCallbacks(TransactionExecutor.java:135)
05-14 18:13:17.572 24318 24318 E AndroidRuntime: 	at android.app.servertransaction.TransactionExecutor.execute(TransactionExecutor.java:95)
05-14 18:13:17.572 24318 24318 E AndroidRuntime: 	at android.app.ActivityThread$H.handleMessage(ActivityThread.java:2438)
05-14 18:13:17.572 24318 24318 E AndroidRuntime: 	at android.os.Handler.dispatchMessage(Handler.java:106)
05-14 18:13:17.572 24318 24318 E AndroidRuntime: 	at android.os.Looper.loopOnce(Looper.java:226)
05-14 18:13:17.572 24318 24318 E AndroidRuntime: 	at android.os.Looper.loop(Looper.java:313)
05-14 18:13:17.572 24318 24318 E AndroidRuntime: 	at android.app.ActivityThread.main(ActivityThread.java:8663)
05-14 18:13:17.572 24318 24318 E AndroidRuntime: 	at java.lang.reflect.Method.invoke(Native Method)
05-14 18:13:17.572 24318 24318 E AndroidRuntime: 	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:567)
05-14 18:13:17.572 24318 24318 E AndroidRuntime: 	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1135)
05-14 18:13:17.572 24318 24318 E AndroidRuntime: Caused by: java.lang.NullPointerException: Attempt to invoke virtual method 'boolean cgeo.geocaching.ui.ImageGalleryView.onActivityResult(int, int, android.content.Intent)' on a null object reference
05-14 18:13:17.572 24318 24318 E AndroidRuntime: 	at cgeo.geocaching.TrackableActivity.onActivityResult(TrackableActivity.java:739)
05-14 18:13:17.572 24318 24318 E AndroidRuntime: 	at android.app.Activity.dispatchActivityResult(Activity.java:8659)
05-14 18:13:17.572 24318 24318 E AndroidRuntime: 	at android.app.ActivityThread.deliverResults(ActivityThread.java:5857)
05-14 18:13:17.572 24318 24318 E AndroidRuntime: 	... 13 more
05-14 18:15:06.875 24442 24442 E AndroidRuntime: FATAL EXCEPTION: main
05-14 18:15:06.875 24442 24442 E AndroidRuntime: Process: cgeo.geocaching, PID: 24442
05-14 18:15:06.875 24442 24442 E AndroidRuntime: java.lang.RuntimeException: Failure delivering result ResultInfo{who=null, request=1, result=0, data=null} to activity {cgeo.geocaching/cgeo.geocaching.TrackableActivity}: java.lang.NullPointerException: Attempt to invoke virtual method 'boolean cgeo.geocaching.ui.ImageGalleryView.onActivityResult(int, int, android.content.Intent)' on a null object reference
05-14 18:15:06.875 24442 24442 E AndroidRuntime: 	at android.app.ActivityThread.deliverResults(ActivityThread.java:5864)
05-14 18:15:06.875 24442 24442 E AndroidRuntime: 	at android.app.ActivityThread.handleSendResult(ActivityThread.java:5903)
05-14 18:15:06.875 24442 24442 E AndroidRuntime: 	at android.app.servertransaction.ActivityResultItem.execute(ActivityResultItem.java:54)
05-14 18:15:06.875 24442 24442 E AndroidRuntime: 	at android.app.servertransaction.ActivityTransactionItem.execute(ActivityTransactionItem.java:45)
05-14 18:15:06.875 24442 24442 E AndroidRuntime: 	at android.app.servertransaction.TransactionExecutor.executeCallbacks(TransactionExecutor.java:135)
05-14 18:15:06.875 24442 24442 E AndroidRuntime: 	at android.app.servertransaction.TransactionExecutor.execute(TransactionExecutor.java:95)
05-14 18:15:06.875 24442 24442 E AndroidRuntime: 	at android.app.ActivityThread$H.handleMessage(ActivityThread.java:2438)
05-14 18:15:06.875 24442 24442 E AndroidRuntime: 	at android.os.Handler.dispatchMessage(Handler.java:106)
05-14 18:15:06.875 24442 24442 E AndroidRuntime: 	at android.os.Looper.loopOnce(Looper.java:226)
05-14 18:15:06.875 24442 24442 E AndroidRuntime: 	at android.os.Looper.loop(Looper.java:313)
05-14 18:15:06.875 24442 24442 E AndroidRuntime: 	at android.app.ActivityThread.main(ActivityThread.java:8663)
05-14 18:15:06.875 24442 24442 E AndroidRuntime: 	at java.lang.reflect.Method.invoke(Native Method)
05-14 18:15:06.875 24442 24442 E AndroidRuntime: 	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:567)
05-14 18:15:06.875 24442 24442 E AndroidRuntime: 	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1135)
05-14 18:15:06.875 24442 24442 E AndroidRuntime: Caused by: java.lang.NullPointerException: Attempt to invoke virtual method 'boolean cgeo.geocaching.ui.ImageGalleryView.onActivityResult(int, int, android.content.Intent)' on a null object reference
05-14 18:15:06.875 24442 24442 E AndroidRuntime: 	at cgeo.geocaching.TrackableActivity.onActivityResult(TrackableActivity.java:739)
05-14 18:15:06.875 24442 24442 E AndroidRuntime: 	at android.app.Activity.dispatchActivityResult(Activity.java:8659)
05-14 18:15:06.875 24442 24442 E AndroidRuntime: 	at android.app.ActivityThread.deliverResults(ActivityThread.java:5857)
05-14 18:15:06.875 24442 24442 E AndroidRuntime: 	... 13 more
05-14 20:55:47.379 30222 30222 E AndroidRuntime: FATAL EXCEPTION: main
05-14 20:55:47.379 30222 30222 E AndroidRuntime: Process: cgeo.geocaching, PID: 30222
05-14 20:55:47.379 30222 30222 E AndroidRuntime: java.lang.RuntimeException: Failure delivering result ResultInfo{who=null, request=1, result=0, data=null} to activity {cgeo.geocaching/cgeo.geocaching.TrackableActivity}: java.lang.NullPointerException: Attempt to invoke virtual method 'boolean cgeo.geocaching.ui.ImageGalleryView.onActivityResult(int, int, android.content.Intent)' on a null object reference
05-14 20:55:47.379 30222 30222 E AndroidRuntime: 	at android.app.ActivityThread.deliverResults(ActivityThread.java:5864)
05-14 20:55:47.379 30222 30222 E AndroidRuntime: 	at android.app.ActivityThread.handleSendResult(ActivityThread.java:5903)
05-14 20:55:47.379 30222 30222 E AndroidRuntime: 	at android.app.servertransaction.ActivityResultItem.execute(ActivityResultItem.java:54)
05-14 20:55:47.379 30222 30222 E AndroidRuntime: 	at android.app.servertransaction.ActivityTransactionItem.execute(ActivityTransactionItem.java:45)
05-14 20:55:47.379 30222 30222 E AndroidRuntime: 	at android.app.servertransaction.TransactionExecutor.executeCallbacks(TransactionExecutor.java:135)
05-14 20:55:47.379 30222 30222 E AndroidRuntime: 	at android.app.servertransaction.TransactionExecutor.execute(TransactionExecutor.java:95)
05-14 20:55:47.379 30222 30222 E AndroidRuntime: 	at android.app.ActivityThread$H.handleMessage(ActivityThread.java:2438)
05-14 20:55:47.379 30222 30222 E AndroidRuntime: 	at android.os.Handler.dispatchMessage(Handler.java:106)
05-14 20:55:47.379 30222 30222 E AndroidRuntime: 	at android.os.Looper.loopOnce(Looper.java:226)
05-14 20:55:47.379 30222 30222 E AndroidRuntime: 	at android.os.Looper.loop(Looper.java:313)
05-14 20:55:47.379 30222 30222 E AndroidRuntime: 	at android.app.ActivityThread.main(ActivityThread.java:8663)
05-14 20:55:47.379 30222 30222 E AndroidRuntime: 	at java.lang.reflect.Method.invoke(Native Method)
05-14 20:55:47.379 30222 30222 E AndroidRuntime: 	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:567)
05-14 20:55:47.379 30222 30222 E AndroidRuntime: 	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1135)
05-14 20:55:47.379 30222 30222 E AndroidRuntime: Caused by: java.lang.NullPointerException: Attempt to invoke virtual method 'boolean cgeo.geocaching.ui.ImageGalleryView.onActivityResult(int, int, android.content.Intent)' on a null object reference
05-14 20:55:47.379 30222 30222 E AndroidRuntime: 	at cgeo.geocaching.TrackableActivity.onActivityResult(TrackableActivity.java:739)
05-14 20:55:47.379 30222 30222 E AndroidRuntime: 	at android.app.Activity.dispatchActivityResult(Activity.java:8659)
05-14 20:55:47.379 30222 30222 E AndroidRuntime: 	at android.app.ActivityThread.deliverResults(ActivityThread.java:5857)
05-14 20:55:47.379 30222 30222 E AndroidRuntime: 	... 13 more
05-14 21:00:31.282 31328 31328 E AndroidRuntime: FATAL EXCEPTION: main
05-14 21:00:31.282 31328 31328 E AndroidRuntime: Process: cgeo.geocaching, PID: 31328
05-14 21:00:31.282 31328 31328 E AndroidRuntime: java.lang.RuntimeException: Failure delivering result ResultInfo{who=null, request=1, result=0, data=null} to activity {cgeo.geocaching/cgeo.geocaching.TrackableActivity}: java.lang.NullPointerException: Attempt to invoke virtual method 'boolean cgeo.geocaching.ui.ImageGalleryView.onActivityResult(int, int, android.content.Intent)' on a null object reference
05-14 21:00:31.282 31328 31328 E AndroidRuntime: 	at android.app.ActivityThread.deliverResults(ActivityThread.java:5864)
05-14 21:00:31.282 31328 31328 E AndroidRuntime: 	at android.app.ActivityThread.handleSendResult(ActivityThread.java:5903)
05-14 21:00:31.282 31328 31328 E AndroidRuntime: 	at android.app.servertransaction.ActivityResultItem.execute(ActivityResultItem.java:54)
05-14 21:00:31.282 31328 31328 E AndroidRuntime: 	at android.app.servertransaction.ActivityTransactionItem.execute(ActivityTransactionItem.java:45)
05-14 21:00:31.282 31328 31328 E AndroidRuntime: 	at android.app.servertransaction.TransactionExecutor.executeCallbacks(TransactionExecutor.java:135)
05-14 21:00:31.282 31328 31328 E AndroidRuntime: 	at android.app.servertransaction.TransactionExecutor.execute(TransactionExecutor.java:95)
05-14 21:00:31.282 31328 31328 E AndroidRuntime: 	at android.app.ActivityThread$H.handleMessage(ActivityThread.java:2438)
05-14 21:00:31.282 31328 31328 E AndroidRuntime: 	at android.os.Handler.dispatchMessage(Handler.java:106)
05-14 21:00:31.282 31328 31328 E AndroidRuntime: 	at android.os.Looper.loopOnce(Looper.java:226)
05-14 21:00:31.282 31328 31328 E AndroidRuntime: 	at android.os.Looper.loop(Looper.java:313)
05-14 21:00:31.282 31328 31328 E AndroidRuntime: 	at android.app.ActivityThread.main(ActivityThread.java:8663)
05-14 21:00:31.282 31328 31328 E AndroidRuntime: 	at java.lang.reflect.Method.invoke(Native Method)
05-14 21:00:31.282 31328 31328 E AndroidRuntime: 	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:567)
05-14 21:00:31.282 31328 31328 E AndroidRuntime: 	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1135)
05-14 21:00:31.282 31328 31328 E AndroidRuntime: Caused by: java.lang.NullPointerException: Attempt to invoke virtual method 'boolean cgeo.geocaching.ui.ImageGalleryView.onActivityResult(int, int, android.content.Intent)' on a null object reference
05-14 21:00:31.282 31328 31328 E AndroidRuntime: 	at cgeo.geocaching.TrackableActivity.onActivityResult(TrackableActivity.java:739)
05-14 21:00:31.282 31328 31328 E AndroidRuntime: 	at android.app.Activity.dispatchActivityResult(Activity.java:8659)
05-14 21:00:31.282 31328 31328 E AndroidRuntime: 	at android.app.ActivityThread.deliverResults(ActivityThread.java:5857)
05-14 21:00:31.282 31328 31328 E AndroidRuntime: 	... 13 more
05-14 21:01:21.149  1847  1847 E AndroidRuntime: FATAL EXCEPTION: main
05-14 21:01:21.149  1847  1847 E AndroidRuntime: Process: cgeo.geocaching, PID: 1847
05-14 21:01:21.149  1847  1847 E AndroidRuntime: java.lang.RuntimeException: Failure delivering result ResultInfo{who=null, request=1, result=0, data=null} to activity {cgeo.geocaching/cgeo.geocaching.TrackableActivity}: java.lang.NullPointerException: Attempt to invoke virtual method 'boolean cgeo.geocaching.ui.ImageGalleryView.onActivityResult(int, int, android.content.Intent)' on a null object reference
05-14 21:01:21.149  1847  1847 E AndroidRuntime: 	at android.app.ActivityThread.deliverResults(ActivityThread.java:5864)
05-14 21:01:21.149  1847  1847 E AndroidRuntime: 	at android.app.ActivityThread.handleSendResult(ActivityThread.java:5903)
05-14 21:01:21.149  1847  1847 E AndroidRuntime: 	at android.app.servertransaction.ActivityResultItem.execute(ActivityResultItem.java:54)
05-14 21:01:21.149  1847  1847 E AndroidRuntime: 	at android.app.servertransaction.ActivityTransactionItem.execute(ActivityTransactionItem.java:45)
05-14 21:01:21.149  1847  1847 E AndroidRuntime: 	at android.app.servertransaction.TransactionExecutor.executeCallbacks(TransactionExecutor.java:135)
05-14 21:01:21.149  1847  1847 E AndroidRuntime: 	at android.app.servertransaction.TransactionExecutor.execute(TransactionExecutor.java:95)
05-14 21:01:21.149  1847  1847 E AndroidRuntime: 	at android.app.ActivityThread$H.handleMessage(ActivityThread.java:2438)
05-14 21:01:21.149  1847  1847 E AndroidRuntime: 	at android.os.Handler.dispatchMessage(Handler.java:106)
05-14 21:01:21.149  1847  1847 E AndroidRuntime: 	at android.os.Looper.loopOnce(Looper.java:226)
05-14 21:01:21.149  1847  1847 E AndroidRuntime: 	at android.os.Looper.loop(Looper.java:313)
05-14 21:01:21.149  1847  1847 E AndroidRuntime: 	at android.app.ActivityThread.main(ActivityThread.java:8663)
05-14 21:01:21.149  1847  1847 E AndroidRuntime: 	at java.lang.reflect.Method.invoke(Native Method)
05-14 21:01:21.149  1847  1847 E AndroidRuntime: 	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:567)
05-14 21:01:21.149  1847  1847 E AndroidRuntime: 	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1135)
05-14 21:01:21.149  1847  1847 E AndroidRuntime: Caused by: java.lang.NullPointerException: Attempt to invoke virtual method 'boolean cgeo.geocaching.ui.ImageGalleryView.onActivityResult(int, int, android.content.Intent)' on a null object reference
05-14 21:01:21.149  1847  1847 E AndroidRuntime: 	at cgeo.geocaching.TrackableActivity.onActivityResult(TrackableActivity.java:739)
05-14 21:01:21.149  1847  1847 E AndroidRuntime: 	at android.app.Activity.dispatchActivityResult(Activity.java:8659)
05-14 21:01:21.149  1847  1847 E AndroidRuntime: 	at android.app.ActivityThread.deliverResults(ActivityThread.java:5857)
05-14 21:01:21.149  1847  1847 E AndroidRuntime: 	... 13 more
05-14 21:01:55.768  3036  3036 E AndroidRuntime: FATAL EXCEPTION: main
05-14 21:01:55.768  3036  3036 E AndroidRuntime: Process: cgeo.geocaching, PID: 3036
05-14 21:01:55.768  3036  3036 E AndroidRuntime: java.lang.RuntimeException: Failure delivering result ResultInfo{who=null, request=1, result=0, data=null} to activity {cgeo.geocaching/cgeo.geocaching.TrackableActivity}: java.lang.NullPointerException: Attempt to invoke virtual method 'boolean cgeo.geocaching.ui.ImageGalleryView.onActivityResult(int, int, android.content.Intent)' on a null object reference
05-14 21:01:55.768  3036  3036 E AndroidRuntime: 	at android.app.ActivityThread.deliverResults(ActivityThread.java:5864)
05-14 21:01:55.768  3036  3036 E AndroidRuntime: 	at android.app.ActivityThread.handleSendResult(ActivityThread.java:5903)
05-14 21:01:55.768  3036  3036 E AndroidRuntime: 	at android.app.servertransaction.ActivityResultItem.execute(ActivityResultItem.java:54)
05-14 21:01:55.768  3036  3036 E AndroidRuntime: 	at android.app.servertransaction.ActivityTransactionItem.execute(ActivityTransactionItem.java:45)
05-14 21:01:55.768  3036  3036 E AndroidRuntime: 	at android.app.servertransaction.TransactionExecutor.executeCallbacks(TransactionExecutor.java:135)
05-14 21:01:55.768  3036  3036 E AndroidRuntime: 	at android.app.servertransaction.TransactionExecutor.execute(TransactionExecutor.java:95)
05-14 21:01:55.768  3036  3036 E AndroidRuntime: 	at android.app.ActivityThread$H.handleMessage(ActivityThread.java:2438)
05-14 21:01:55.768  3036  3036 E AndroidRuntime: 	at android.os.Handler.dispatchMessage(Handler.java:106)
05-14 21:01:55.768  3036  3036 E AndroidRuntime: 	at android.os.Looper.loopOnce(Looper.java:226)
05-14 21:01:55.768  3036  3036 E AndroidRuntime: 	at android.os.Looper.loop(Looper.java:313)
05-14 21:01:55.768  3036  3036 E AndroidRuntime: 	at android.app.ActivityThread.main(ActivityThread.java:8663)
05-14 21:01:55.768  3036  3036 E AndroidRuntime: 	at java.lang.reflect.Method.invoke(Native Method)
05-14 21:01:55.768  3036  3036 E AndroidRuntime: 	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:567)
05-14 21:01:55.768  3036  3036 E AndroidRuntime: 	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1135)
05-14 21:01:55.768  3036  3036 E AndroidRuntime: Caused by: java.lang.NullPointerException: Attempt to invoke virtual method 'boolean cgeo.geocaching.ui.ImageGalleryView.onActivityResult(int, int, android.content.Intent)' on a null object reference
05-14 21:01:55.768  3036  3036 E AndroidRuntime: 	at cgeo.geocaching.TrackableActivity.onActivityResult(TrackableActivity.java:739)
05-14 21:01:55.768  3036  3036 E AndroidRuntime: 	at android.app.Activity.dispatchActivityResult(Activity.java:8659)
05-14 21:01:55.768  3036  3036 E AndroidRuntime: 	at android.app.ActivityThread.deliverResults(ActivityThread.java:5857)
05-14 21:01:55.768  3036  3036 E AndroidRuntime: 	... 13 more
05-14 21:02:25.880  3519  3519 E AndroidRuntime: FATAL EXCEPTION: main
05-14 21:02:25.880  3519  3519 E AndroidRuntime: Process: cgeo.geocaching, PID: 3519
05-14 21:02:25.880  3519  3519 E AndroidRuntime: java.lang.RuntimeException: Failure delivering result ResultInfo{who=null, request=1, result=0, data=null} to activity {cgeo.geocaching/cgeo.geocaching.TrackableActivity}: java.lang.NullPointerException: Attempt to invoke virtual method 'boolean cgeo.geocaching.ui.ImageGalleryView.onActivityResult(int, int, android.content.Intent)' on a null object reference
05-14 21:02:25.880  3519  3519 E AndroidRuntime: 	at android.app.ActivityThread.deliverResults(ActivityThread.java:5864)
05-14 21:02:25.880  3519  3519 E AndroidRuntime: 	at android.app.ActivityThread.handleSendResult(ActivityThread.java:5903)
05-14 21:02:25.880  3519  3519 E AndroidRuntime: 	at android.app.servertransaction.ActivityResultItem.execute(ActivityResultItem.java:54)
05-14 21:02:25.880  3519  3519 E AndroidRuntime: 	at android.app.servertransaction.ActivityTransactionItem.execute(ActivityTransactionItem.java:45)
05-14 21:02:25.880  3519  3519 E AndroidRuntime: 	at android.app.servertransaction.TransactionExecutor.executeCallbacks(TransactionExecutor.java:135)
05-14 21:02:25.880  3519  3519 E AndroidRuntime: 	at android.app.servertransaction.TransactionExecutor.execute(TransactionExecutor.java:95)
05-14 21:02:25.880  3519  3519 E AndroidRuntime: 	at android.app.ActivityThread$H.handleMessage(ActivityThread.java:2438)
05-14 21:02:25.880  3519  3519 E AndroidRuntime: 	at android.os.Handler.dispatchMessage(Handler.java:106)
05-14 21:02:25.880  3519  3519 E AndroidRuntime: 	at android.os.Looper.loopOnce(Looper.java:226)
05-14 21:02:25.880  3519  3519 E AndroidRuntime: 	at android.os.Looper.loop(Looper.java:313)
05-14 21:02:25.880  3519  3519 E AndroidRuntime: 	at android.app.ActivityThread.main(ActivityThread.java:8663)
05-14 21:02:25.880  3519  3519 E AndroidRuntime: 	at java.lang.reflect.Method.invoke(Native Method)
05-14 21:02:25.880  3519  3519 E AndroidRuntime: 	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:567)
05-14 21:02:25.880  3519  3519 E AndroidRuntime: 	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1135)
05-14 21:02:25.880  3519  3519 E AndroidRuntime: Caused by: java.lang.NullPointerException: Attempt to invoke virtual method 'boolean cgeo.geocaching.ui.ImageGalleryView.onActivityResult(int, int, android.content.Intent)' on a null object reference
05-14 21:02:25.880  3519  3519 E AndroidRuntime: 	at cgeo.geocaching.TrackableActivity.onActivityResult(TrackableActivity.java:739)
05-14 21:02:25.880  3519  3519 E AndroidRuntime: 	at android.app.Activity.dispatchActivityResult(Activity.java:8659)
05-14 21:02:25.880  3519  3519 E AndroidRuntime: 	at android.app.ActivityThread.deliverResults(ActivityThread.java:5857)
05-14 21:02:25.880  3519  3519 E AndroidRuntime: 	... 13 more
05-14 21:03:26.585  4017  4017 E AndroidRuntime: FATAL EXCEPTION: main
05-14 21:03:26.585  4017  4017 E AndroidRuntime: Process: cgeo.geocaching, PID: 4017
05-14 21:03:26.585  4017  4017 E AndroidRuntime: java.lang.RuntimeException: Failure delivering result ResultInfo{who=null, request=1, result=0, data=null} to activity {cgeo.geocaching/cgeo.geocaching.TrackableActivity}: java.lang.NullPointerException: Attempt to invoke virtual method 'boolean cgeo.geocaching.ui.ImageGalleryView.onActivityResult(int, int, android.content.Intent)' on a null object reference
05-14 21:03:26.585  4017  4017 E AndroidRuntime: 	at android.app.ActivityThread.deliverResults(ActivityThread.java:5864)
05-14 21:03:26.585  4017  4017 E AndroidRuntime: 	at android.app.ActivityThread.handleSendResult(ActivityThread.java:5903)
05-14 21:03:26.585  4017  4017 E AndroidRuntime: 	at android.app.servertransaction.ActivityResultItem.execute(ActivityResultItem.java:54)
05-14 21:03:26.585  4017  4017 E AndroidRuntime: 	at android.app.servertransaction.ActivityTransactionItem.execute(ActivityTransactionItem.java:45)
05-14 21:03:26.585  4017  4017 E AndroidRuntime: 	at android.app.servertransaction.TransactionExecutor.executeCallbacks(TransactionExecutor.java:135)
05-14 21:03:26.585  4017  4017 E AndroidRuntime: 	at android.app.servertransaction.TransactionExecutor.execute(TransactionExecutor.java:95)
05-14 21:03:26.585  4017  4017 E AndroidRuntime: 	at android.app.ActivityThread$H.handleMessage(ActivityThread.java:2438)
05-14 21:03:26.585  4017  4017 E AndroidRuntime: 	at android.os.Handler.dispatchMessage(Handler.java:106)
05-14 21:03:26.585  4017  4017 E AndroidRuntime: 	at android.os.Looper.loopOnce(Looper.java:226)
05-14 21:03:26.585  4017  4017 E AndroidRuntime: 	at android.os.Looper.loop(Looper.java:313)
05-14 21:03:26.585  4017  4017 E AndroidRuntime: 	at android.app.ActivityThread.main(ActivityThread.java:8663)
05-14 21:03:26.585  4017  4017 E AndroidRuntime: 	at java.lang.reflect.Method.invoke(Native Method)
05-14 21:03:26.585  4017  4017 E AndroidRuntime: 	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:567)
05-14 21:03:26.585  4017  4017 E AndroidRuntime: 	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1135)
05-14 21:03:26.585  4017  4017 E AndroidRuntime: Caused by: java.lang.NullPointerException: Attempt to invoke virtual method 'boolean cgeo.geocaching.ui.ImageGalleryView.onActivityResult(int, int, android.content.Intent)' on a null object reference
05-14 21:03:26.585  4017  4017 E AndroidRuntime: 	at cgeo.geocaching.TrackableActivity.onActivityResult(TrackableActivity.java:739)
05-14 21:03:26.585  4017  4017 E AndroidRuntime: 	at android.app.Activity.dispatchActivityResult(Activity.java:8659)
05-14 21:03:26.585  4017  4017 E AndroidRuntime: 	at android.app.ActivityThread.deliverResults(ActivityThread.java:5857)
05-14 21:03:26.585  4017  4017 E AndroidRuntime: 	... 13 more
05-14 21:04:36.007  4608  4608 E AndroidRuntime: FATAL EXCEPTION: main
05-14 21:04:36.007  4608  4608 E AndroidRuntime: Process: cgeo.geocaching, PID: 4608
05-14 21:04:36.007  4608  4608 E AndroidRuntime: java.lang.RuntimeException: Failure delivering result ResultInfo{who=null, request=1, result=0, data=null} to activity {cgeo.geocaching/cgeo.geocaching.TrackableActivity}: java.lang.NullPointerException: Attempt to invoke virtual method 'boolean cgeo.geocaching.ui.ImageGalleryView.onActivityResult(int, int, android.content.Intent)' on a null object reference
05-14 21:04:36.007  4608  4608 E AndroidRuntime: 	at android.app.ActivityThread.deliverResults(ActivityThread.java:5864)
05-14 21:04:36.007  4608  4608 E AndroidRuntime: 	at android.app.ActivityThread.handleSendResult(ActivityThread.java:5903)
05-14 21:04:36.007  4608  4608 E AndroidRuntime: 	at android.app.servertransaction.ActivityResultItem.execute(ActivityResultItem.java:54)
05-14 21:04:36.007  4608  4608 E AndroidRuntime: 	at android.app.servertransaction.ActivityTransactionItem.execute(ActivityTransactionItem.java:45)
05-14 21:04:36.007  4608  4608 E AndroidRuntime: 	at android.app.servertransaction.TransactionExecutor.executeCallbacks(TransactionExecutor.java:135)
05-14 21:04:36.007  4608  4608 E AndroidRuntime: 	at android.app.servertransaction.TransactionExecutor.execute(TransactionExecutor.java:95)
05-14 21:04:36.007  4608  4608 E AndroidRuntime: 	at android.app.ActivityThread$H.handleMessage(ActivityThread.java:2438)
05-14 21:04:36.007  4608  4608 E AndroidRuntime: 	at android.os.Handler.dispatchMessage(Handler.java:106)
05-14 21:04:36.007  4608  4608 E AndroidRuntime: 	at android.os.Looper.loopOnce(Looper.java:226)
05-14 21:04:36.007  4608  4608 E AndroidRuntime: 	at android.os.Looper.loop(Looper.java:313)
05-14 21:04:36.007  4608  4608 E AndroidRuntime: 	at android.app.ActivityThread.main(ActivityThread.java:8663)
05-14 21:04:36.007  4608  4608 E AndroidRuntime: 	at java.lang.reflect.Method.invoke(Native Method)
05-14 21:04:36.007  4608  4608 E AndroidRuntime: 	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:567)
05-14 21:04:36.007  4608  4608 E AndroidRuntime: 	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1135)
05-14 21:04:36.007  4608  4608 E AndroidRuntime: Caused by: java.lang.NullPointerException: Attempt to invoke virtual method 'boolean cgeo.geocaching.ui.ImageGalleryView.onActivityResult(int, int, android.content.Intent)' on a null object reference
05-14 21:04:36.007  4608  4608 E AndroidRuntime: 	at cgeo.geocaching.TrackableActivity.onActivityResult(TrackableActivity.java:739)
05-14 21:04:36.007  4608  4608 E AndroidRuntime: 	at android.app.Activity.dispatchActivityResult(Activity.java:8659)
05-14 21:04:36.007  4608  4608 E AndroidRuntime: 	at android.app.ActivityThread.deliverResults(ActivityThread.java:5857)
05-14 21:04:36.007  4608  4608 E AndroidRuntime: 	... 13 more
05-14 21:05:53.229  5625  5625 E AndroidRuntime: FATAL EXCEPTION: main
05-14 21:05:53.229  5625  5625 E AndroidRuntime: Process: cgeo.geocaching, PID: 5625
05-14 21:05:53.229  5625  5625 E AndroidRuntime: java.lang.RuntimeException: Failure delivering result ResultInfo{who=null, request=1, result=0, data=null} to activity {cgeo.geocaching/cgeo.geocaching.TrackableActivity}: java.lang.NullPointerException: Attempt to invoke virtual method 'boolean cgeo.geocaching.ui.ImageGalleryView.onActivityResult(int, int, android.content.Intent)' on a null object reference
05-14 21:05:53.229  5625  5625 E AndroidRuntime: 	at android.app.ActivityThread.deliverResults(ActivityThread.java:5864)
05-14 21:05:53.229  5625  5625 E AndroidRuntime: 	at android.app.ActivityThread.handleSendResult(ActivityThread.java:5903)
05-14 21:05:53.229  5625  5625 E AndroidRuntime: 	at android.app.servertransaction.ActivityResultItem.execute(ActivityResultItem.java:54)
05-14 21:05:53.229  5625  5625 E AndroidRuntime: 	at android.app.servertransaction.ActivityTransactionItem.execute(ActivityTransactionItem.java:45)
05-14 21:05:53.229  5625  5625 E AndroidRuntime: 	at android.app.servertransaction.TransactionExecutor.executeCallbacks(TransactionExecutor.java:135)
05-14 21:05:53.229  5625  5625 E AndroidRuntime: 	at android.app.servertransaction.TransactionExecutor.execute(TransactionExecutor.java:95)
05-14 21:05:53.229  5625  5625 E AndroidRuntime: 	at android.app.ActivityThread$H.handleMessage(ActivityThread.java:2438)
05-14 21:05:53.229  5625  5625 E AndroidRuntime: 	at android.os.Handler.dispatchMessage(Handler.java:106)
05-14 21:05:53.229  5625  5625 E AndroidRuntime: 	at android.os.Looper.loopOnce(Looper.java:226)
05-14 21:05:53.229  5625  5625 E AndroidRuntime: 	at android.os.Looper.loop(Looper.java:313)
05-14 21:05:53.229  5625  5625 E AndroidRuntime: 	at android.app.ActivityThread.main(ActivityThread.java:8663)
05-14 21:05:53.229  5625  5625 E AndroidRuntime: 	at java.lang.reflect.Method.invoke(Native Method)
05-14 21:05:53.229  5625  5625 E AndroidRuntime: 	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:567)
05-14 21:05:53.229  5625  5625 E AndroidRuntime: 	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1135)
05-14 21:05:53.229  5625  5625 E AndroidRuntime: Caused by: java.lang.NullPointerException: Attempt to invoke virtual method 'boolean cgeo.geocaching.ui.ImageGalleryView.onActivityResult(int, int, android.content.Intent)' on a null object reference
05-14 21:05:53.229  5625  5625 E AndroidRuntime: 	at cgeo.geocaching.TrackableActivity.onActivityResult(TrackableActivity.java:739)
05-14 21:05:53.229  5625  5625 E AndroidRuntime: 	at android.app.Activity.dispatchActivityResult(Activity.java:8659)
05-14 21:05:53.229  5625  5625 E AndroidRuntime: 	at android.app.ActivityThread.deliverResults(ActivityThread.java:5857)
05-14 21:05:53.229  5625  5625 E AndroidRuntime: 	... 13 more
05-14 21:06:52.316  6864  6864 E AndroidRuntime: FATAL EXCEPTION: main
05-14 21:06:52.316  6864  6864 E AndroidRuntime: Process: cgeo.geocaching, PID: 6864
05-14 21:06:52.316  6864  6864 E AndroidRuntime: java.lang.RuntimeException: Failure delivering result ResultInfo{who=null, request=1, result=0, data=null} to activity {cgeo.geocaching/cgeo.geocaching.TrackableActivity}: java.lang.NullPointerException: Attempt to invoke virtual method 'boolean cgeo.geocaching.ui.ImageGalleryView.onActivityResult(int, int, android.content.Intent)' on a null object reference
05-14 21:06:52.316  6864  6864 E AndroidRuntime: 	at android.app.ActivityThread.deliverResults(ActivityThread.java:5864)
05-14 21:06:52.316  6864  6864 E AndroidRuntime: 	at android.app.ActivityThread.handleSendResult(ActivityThread.java:5903)
05-14 21:06:52.316  6864  6864 E AndroidRuntime: 	at android.app.servertransaction.ActivityResultItem.execute(ActivityResultItem.java:54)
05-14 21:06:52.316  6864  6864 E AndroidRuntime: 	at android.app.servertransaction.ActivityTransactionItem.execute(ActivityTransactionItem.java:45)
05-14 21:06:52.316  6864  6864 E AndroidRuntime: 	at android.app.servertransaction.TransactionExecutor.executeCallbacks(TransactionExecutor.java:135)
05-14 21:06:52.316  6864  6864 E AndroidRuntime: 	at android.app.servertransaction.TransactionExecutor.execute(TransactionExecutor.java:95)
05-14 21:06:52.316  6864  6864 E AndroidRuntime: 	at android.app.ActivityThread$H.handleMessage(ActivityThread.java:2438)
05-14 21:06:52.316  6864  6864 E AndroidRuntime: 	at android.os.Handler.dispatchMessage(Handler.java:106)
05-14 21:06:52.316  6864  6864 E AndroidRuntime: 	at android.os.Looper.loopOnce(Looper.java:226)
05-14 21:06:52.316  6864  6864 E AndroidRuntime: 	at android.os.Looper.loop(Looper.java:313)
05-14 21:06:52.316  6864  6864 E AndroidRuntime: 	at android.app.ActivityThread.main(ActivityThread.java:8663)
05-14 21:06:52.316  6864  6864 E AndroidRuntime: 	at java.lang.reflect.Method.invoke(Native Method)
05-14 21:06:52.316  6864  6864 E AndroidRuntime: 	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:567)
05-14 21:06:52.316  6864  6864 E AndroidRuntime: 	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1135)
05-14 21:06:52.316  6864  6864 E AndroidRuntime: Caused by: java.lang.NullPointerException: Attempt to invoke virtual method 'boolean cgeo.geocaching.ui.ImageGalleryView.onActivityResult(int, int, android.content.Intent)' on a null object reference
05-14 21:06:52.316  6864  6864 E AndroidRuntime: 	at cgeo.geocaching.TrackableActivity.onActivityResult(TrackableActivity.java:739)
05-14 21:06:52.316  6864  6864 E AndroidRuntime: 	at android.app.Activity.dispatchActivityResult(Activity.java:8659)
05-14 21:06:52.316  6864  6864 E AndroidRuntime: 	at android.app.ActivityThread.deliverResults(ActivityThread.java:5857)
05-14 21:06:52.316  6864  6864 E AndroidRuntime: 	... 13 more
05-14 21:07:50.514  7721  7721 E AndroidRuntime: FATAL EXCEPTION: main
05-14 21:07:50.514  7721  7721 E AndroidRuntime: Process: cgeo.geocaching, PID: 7721
05-14 21:07:50.514  7721  7721 E AndroidRuntime: java.lang.RuntimeException: Failure delivering result ResultInfo{who=null, request=1, result=0, data=null} to activity {cgeo.geocaching/cgeo.geocaching.TrackableActivity}: java.lang.NullPointerException: Attempt to invoke virtual method 'boolean cgeo.geocaching.ui.ImageGalleryView.onActivityResult(int, int, android.content.Intent)' on a null object reference
05-14 21:07:50.514  7721  7721 E AndroidRuntime: 	at android.app.ActivityThread.deliverResults(ActivityThread.java:5864)
05-14 21:07:50.514  7721  7721 E AndroidRuntime: 	at android.app.ActivityThread.handleSendResult(ActivityThread.java:5903)
05-14 21:07:50.514  7721  7721 E AndroidRuntime: 	at android.app.servertransaction.ActivityResultItem.execute(ActivityResultItem.java:54)
05-14 21:07:50.514  7721  7721 E AndroidRuntime: 	at android.app.servertransaction.ActivityTransactionItem.execute(ActivityTransactionItem.java:45)
05-14 21:07:50.514  7721  7721 E AndroidRuntime: 	at android.app.servertransaction.TransactionExecutor.executeCallbacks(TransactionExecutor.java:135)
05-14 21:07:50.514  7721  7721 E AndroidRuntime: 	at android.app.servertransaction.TransactionExecutor.execute(TransactionExecutor.java:95)
05-14 21:07:50.514  7721  7721 E AndroidRuntime: 	at android.app.ActivityThread$H.handleMessage(ActivityThread.java:2438)
05-14 21:07:50.514  7721  7721 E AndroidRuntime: 	at android.os.Handler.dispatchMessage(Handler.java:106)
05-14 21:07:50.514  7721  7721 E AndroidRuntime: 	at android.os.Looper.loopOnce(Looper.java:226)
05-14 21:07:50.514  7721  7721 E AndroidRuntime: 	at android.os.Looper.loop(Looper.java:313)
05-14 21:07:50.514  7721  7721 E AndroidRuntime: 	at android.app.ActivityThread.main(ActivityThread.java:8663)
05-14 21:07:50.514  7721  7721 E AndroidRuntime: 	at java.lang.reflect.Method.invoke(Native Method)
05-14 21:07:50.514  7721  7721 E AndroidRuntime: 	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:567)
05-14 21:07:50.514  7721  7721 E AndroidRuntime: 	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1135)
05-14 21:07:50.514  7721  7721 E AndroidRuntime: Caused by: java.lang.NullPointerException: Attempt to invoke virtual method 'boolean cgeo.geocaching.ui.ImageGalleryView.onActivityResult(int, int, android.content.Intent)' on a null object reference
05-14 21:07:50.514  7721  7721 E AndroidRuntime: 	at cgeo.geocaching.TrackableActivity.onActivityResult(TrackableActivity.java:739)
05-14 21:07:50.514  7721  7721 E AndroidRuntime: 	at android.app.Activity.dispatchActivityResult(Activity.java:8659)
05-14 21:07:50.514  7721  7721 E AndroidRuntime: 	at android.app.ActivityThread.deliverResults(ActivityThread.java:5857)
05-14 21:07:50.514  7721  7721 E AndroidRuntime: 	... 13 more
```

--- a/main/src/cgeo/geocaching/ImageViewActivity.java
+++ b/main/src/cgeo/geocaching/ImageViewActivity.java
@@ -470,9 +470,11 @@ public class ImageViewActivity extends AbstractActionBarActivity {
                     }
                     final int pos = Integer.parseInt(names.get(0).substring(TRANSITION_ID_BACK.length()));
                     final View targetView = getImageView.call(pos);
-                    targetView.setTransitionName(names.get(0));
-                    sharedElements.clear();
-                    sharedElements.put(names.get(0), targetView);
+                    if (targetView != null) {
+                        targetView.setTransitionName(names.get(0));
+                        sharedElements.clear();
+                        sharedElements.put(names.get(0), targetView);
+                    }
                 }
             });
             final View posImageView = getImageView.call(pos);

--- a/main/src/cgeo/geocaching/TrackableActivity.java
+++ b/main/src/cgeo/geocaching/TrackableActivity.java
@@ -735,7 +735,7 @@ public class TrackableActivity extends TabbedViewPagerActivity implements Androi
         // Refresh the logs view after coming back from logging a trackable
         if (requestCode == LogTrackableActivity.LOG_TRACKABLE && resultCode == RESULT_OK) {
             refreshTrackable(StringUtils.defaultIfBlank(trackable.getName(), trackable.getGeocode()));
-        } else {
+        } else if (imageGallery != null) {
             imageGallery.onActivityResult(requestCode, resultCode, data);
         }
     }

--- a/main/src/cgeo/geocaching/ui/ImageGalleryView.java
+++ b/main/src/cgeo/geocaching/ui/ImageGalleryView.java
@@ -409,7 +409,7 @@ public class ImageGalleryView extends LinearLayout {
             view.requestRectangleOnScreen(rect, true);
         }
 
-        return view.findViewById(R.id.image_image);
+        return view == null ? null : view.findViewById(R.id.image_image);
     }
 
     public void clear() {

--- a/main/src/cgeo/geocaching/utils/BackupUtils.java
+++ b/main/src/cgeo/geocaching/utils/BackupUtils.java
@@ -737,7 +737,7 @@ public class BackupUtils {
             Log.i("no old backups to remove");
             return null;
         }
-        Log.e("old backups to remove: " + dirs);
+        Log.w("old backups to remove: " + dirs);
         return dirs.subList(0, dirs.size() - maxBackupNumber);
     }
 

